docs/signature-layout.md

```diff
@@ -0,0 +1,29 @@
+# Signature File Layout
+
+A common file layout for storing and serving signatures provides a consistent way to reference image signatures. Signatures on a filesystem or a web server shall use this common layout. Signatures stored in a REST API are not required to use this common layout.
+
+## Specification
+
+**SCHEME://[URI]/PATH_PREFIX/REGISTRY/REPOSITORY/IMAGE@MANIFEST_DIGEST/signature-INT**
+
+* **SCHEME**: The transport scheme, e.g. **file://** or **https://**
+* **URI**: For remote transport schemes, the signature server URI, e.g. **example.com**
+* **PATH_PREFIX**: The path to the base of the registry directory
+* **REGISTRY**: The registry URI and optional port, e.g. **registry.example.com:5000**
+* **REPOSITORY**: The repository namespace. May occur multiple times for registries that support multiple repository namespaces.
+* **IMAGE**: The name of the image
```

Comment:
Some registries only accept the "namespaceless" repository name, without any default namespace. i.e. pulling REGISTRY/NAME works, but pulling REGISTRY/library/NAME does not.